### PR TITLE
HCP Form Data Collection Improvements Front-end

### PR DESCRIPTION
**Pull request by jamesjypark**
This PR added "Facility Region", "Contact Email Address", and "Additional Notes" input fields to the HCP request form.
The POST object structure was also changed as outlined [here](https://www.qa.bcc3d.ca/api/v1/apidocs/).

Since the front-end changes have only been tested locally, we will want to test it in QA before deploying the changes to PROD.

Below are screenshots of the UI change.
#### Added "Facility Region" dropdown and "Contact Email Address" input field

Screen Shot 2020-04-12 at 7 43 52 PM
#### Added "Additional Notes" input field
Screen Shot 2020-04-12 at 7 44 02 PM

### CODE REVIEW

File: apps/bcc3d.ca/assets/js/request.js [hunk 1]
```diff
@@ -1,29 +1,60 @@
 ---
 ---
+const REGIONS_API_ENDPOINT = "https://www.qa.bcc3d.ca/api/v1/region";
```
Comment (wsonnex) [inline apps/bcc3d.ca/assets/js/request.js:3]: This should be templated, like `{{ site.endpoints.hcp_submit }}`

File: apps/bcc3d.ca/assets/js/request.js [hunk 2]
```diff
@@ -1,29 +1,60 @@
...
+
+window.onload = () => {
+    fetch(REGIONS_API_ENDPOINT)
+        .then((response) => {
+            return response.json();
```
Comment (wsonnex) [inline apps/bcc3d.ca/assets/js/request.js:8]: lambda with a single return statement can have the braces and return omitted, e.g.
`.then(response => response.json())`

File: apps/bcc3d.ca/assets/js/request.js (hunk 1 above)
Comment (jamesjypark) [reply apps/bcc3d.ca/assets/js/request.js:3]: Where do we keep reference of these endpoints?

Comment (wsonnex) [reply apps/bcc3d.ca/assets/js/request.js]: `config.yml` at the top-level directory, then when you run `sh build.sh` it templates these values into `apps/bcc3d.ca/_config.yml` which is read by the jekyll build process

Comment (wsonnex) [reply apps/bcc3d.ca/assets/js/request.js]: have a look at `templates/apps/bcc3d.ca/_config.yml`, anything in the `templates` folder will have the values in `config.yml` (the top level one) interpolated into it as the first part of the build process in `build.sh`

**Comment by chaosaffe** [comment]
I was unable to run `build.sh` to test the patch without error, but it appears to be a local error. Can someone else build and test?

File: apps/bcc3d.ca/assets/js/request.js (hunk 1 above)
Comment (sixthgear) [reply apps/bcc3d.ca/assets/js/request.js]: My blocker on testing this was the fact that on the local build the flask API endpoints were returning 404s, potentially due to either the nginx proxy_pass directive or the flask app configuration.

I will try again today and report any issues I see. I'll fix up the API code if needed (and if I can figure it out)

**Comment by chaosaffe** [comment]
LGTM

**Pull request merged**